packages/core/src/identity/Identity.spec.ts

```diff
@@ -65,7 +75,9 @@ describe('Identity', () => {
   })
 
   it('should have different keys for signing and boxing', () => {
-    const alice = Identity.buildFromMnemonic(Identity.generateMnemonic())
+    const alice = Identity.buildFromMnemonic(Identity.generateMnemonic(), {
+      signingKeyPairType: 'ed25519',
+    })
     expect(Crypto.coToUInt8(alice.signPublicKeyAsHex)).not.toEqual(
       alice.boxKeyPair.publicKey
     )
```

Comment:
Should we maybe add some SR25519 tests?

Reply:
I didn't really touch tests but maybe as the minimum requirement we may want to test signing and encryption with all three key types.
Everything else I'd say will be done as part of the bigger outstanding test refactor.

Reply:
yes, just the basics here